### ISSUE TITLE
change how questions look on each book page

### ISSUE THREAD

**Issue by meagansiu** [maltyeva/mindimp-pg#37]
After adding all the questions for a few books, it appears to me that the page starts to look cluttered. Is it possible to have links or tabs on each book page that will take students to the questions for each week, instead of having the questions on the front page for each book? I think this could help students better keep track of their assignments for each week, and prevent any book "spoilers".

**Comment by maltyeva** [comment]
yes, definitely doable. Could you provide an example of how you think a good layout should look like? 

Assigning to meagan for now.
